app/policies/grant_policy.rb

```diff
@@ -1,13 +1,9 @@
 # frozen_string_literal: true
 
 class GrantPolicy < AccessPolicy
```

Comment:
This shouldn't inherit from AccessPolicy if you have deleted that file.

Reply:
Updated in latest commit.